actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -5,7 +5,7 @@ module ActionView
   # = Action View Tag Helpers
   module Helpers #:nodoc:
     # Provides methods to generate HTML tags programmatically when you can't use
-    # a Builder. By default, they output XHTML compliant tags.
+    # a Builder.
```

Comment:
Let's swap `when you can't use a Builder` with `both as a modern HTML5 compliant builder style and legacy XHTML compliant tags.`